core/src/main/java/hudson/security/TokenBasedRememberMeServices2.java

```diff
@@ -150,7 +150,7 @@ public void loginSuccess(HttpServletRequest request, HttpServletResponse respons
 
 		String signatureValue = makeTokenSignature(expiryTime, (UserDetails)successfulAuthentication.getPrincipal());
 		String tokenValue = username + ":" + expiryTime + ":" + signatureValue;
-		String tokenValueBase64 = new String(Base64.encodeBase64(tokenValue.getBytes()));
+		String tokenValueBase64 = Base64.getMimeEncoder().encodeToString(tokenValue.getBytes());
```

Comment:
This bit is a little strange to me as the `findRememberMeCookieValue` function uses the regular decoder instead of the Mime decoder